tools/api-markdown-documenter/src/api-item-transforms/configuration/Configuration.ts

```diff
@@ -55,9 +57,30 @@ export interface ApiItemTransformationConfigurationBase {
  */
 export interface ApiItemTransformationConfiguration
 	extends ApiItemTransformationConfigurationBase,
-		ApiItemTransformations,
 		Required<DocumentationSuiteOptions>,
-		Required<LoggingConfiguration> {}
+		Required<LoggingConfiguration> {
+	/**
+	 * {@inheritDoc ApiItemTransformations}
+	 */
+	readonly transformations: ApiItemTransformations;
+
+	/**
+	 * Generates the default section layout used by all default {@link ApiItemTransformations}.
+	 *
+	 * @remarks
+	 *
+	 * Can be used to uniformly control the default output layout for all API item kinds.
+	 *
+	 * API item kind-specific details are passed in, and can be displayed as desired.
+	 *
+	 * @returns The list of {@link SectionNode}s that comprise the top-level section body for the API item.
+	 */
+	readonly defaultSectionLayout: (
```

Comment:
Extracted from `ApiItemTransformations`, since it really isn't a "transformation"; it's a utility _used by_ transformations. It will also make it easier to convert `ApiItemTransformations` to a mapped type in the future.